app/models/landlord.rb

```diff
@@ -3,4 +3,10 @@ class Landlord < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :rental_units
+
+  validates :email, presence: true
+  validates :email, format: { with: /.+@.+\..+/i,
+                             message: "Please enter a valid email address."}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.